src/gd_kotlin.h

```diff
@@ -57,6 +59,10 @@ class GDKotlin {
     KtClass* find_class(const StringName& p_script_path);
 
     const GdKotlinConfiguration& get_configuration();
+
+    bool is_initialized;
+
+    Vector<Pair<String, String>> configuration_errors;
```

Comment:
I would prefer a getter and make those variables private so that it cannot be changed from outside of `GDKotlin`.